pkg/controllermanager/controller/project/project_activity_control_test.go

```diff
@@ -127,60 +124,65 @@ var _ = Describe("Project Activity Reconcile", func() {
 			}
 			return apierrors.NewNotFound(gardencorev1beta1.Resource("Project"), "<unknown>")
 		})
+	})
 
-		k8sGardenRuntimeClient.EXPECT().List(ctx, gomock.AssignableToTypeOf(&gardencorev1beta1.ShootList{}), gomock.Any()).DoAndReturn(func(_ context.Context, obj *gardencorev1beta1.ShootList, _ ...client.ListOption) error {
-			obj.Items = []gardencorev1beta1.Shoot{*shoot, *shootWithoutProject, *errorShoot}
+	Describe("Successful LastActivityTimestamp updates", func() {
+		BeforeEach(func() {
+			k8sGardenRuntimeClient.EXPECT().Patch(ctx, gomock.AssignableToTypeOf(&gardencorev1beta1.Project{}), gomock.Any()).DoAndReturn(
+				func(_ context.Context, prj *gardencorev1beta1.Project, _ client.Patch, _ ...client.PatchOption) error {
+					*project = *prj
+					return nil
+				},
+			).AnyTimes()
+			k8sGardenRuntimeClient.EXPECT().Status().Return(k8sGardenRuntimeClient).AnyTimes()
+		})
 
-			return nil
+		It("Should update the creation timestamp", func() {
```

Comment:
```suggestion
		It("should update the creation timestamp", func() {
```

You can use lowercase everywhere.